Resources/Prototypes/Roles/Jobs/Science/research_director.yml

```diff
@@ -5,8 +5,11 @@
   setPreference: false
   playTimeTracker: JobResearchDirector
   requirements:
+    - !type:DepartmentTimeRequirement
+      department: Epistemics
+      time: 18000
     - !type:OverallPlaytimeRequirement
-      time: 108000
+      time: 86400
```

Comment:
We should make them require *at least a bit* of Epistemologist. Like, so you can't just grind Chaplain :P

Reply:
research director

Reply:
Scientist, then, whatever.

Reply:
chaplain requires epistemics time already